common/app/model/meta.scala

```diff
@@ -911,7 +903,6 @@ final case class Tags(tags: List[Tag]) {
         ),
       ),
       ("blogIds", JsString(blogs.map(_.id).mkString(","))),
-      ("commissioningDesks", JsString(commissioningDesks.mkString(","))),
```

Comment:
Are you sure this is only relevant because to GA? It seems like it could be used by other first or third party scripts.

Reply:
GA might be using it but you are right it might be used elsewhere so I added it back the code in this file 491f0d916a0c9ebdfcfe5a5e5ad122cc5137b9a1 except for `normalisedContentType` which I think it could be used not just for GA as well?